docker/docker-compose-dev-auth.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.3"
+services:
+  sume-auth:
```

Comment:
I think u can use the docker-compose file, no needed to create another

Reply:
ok, I gonna move it there...

Reply:
Deal, moved to the main docker-compose